two-factor.php

```diff
@@ -139,6 +139,23 @@ function wpcom_vip_enforce_two_factor_plugin() {
 			return $limited;
 		}, 9 );
 
+		// Calcuate two factor authentication support outside map_meta_cap to avoid callback loop
+		// see: https://github.com/Automattic/vip-go-mu-plugins/pull/1445#issuecomment-592124810
+		$is_user_using_two_factor = false;
+		
+		// We load the two-factor plugin using wpcom_vip_load_plugin but that skips when skip-plugins is set.
+		// This filter still runs even if skip-plugins is set, so add a check to make sure Two_Factor_Core exists
+		if ( class_exists( 'Two_Factor_Core' ) ) {
+			$is_user_using_two_factor = Two_Factor_Core::is_user_using_two_factor();
+		}
```

Comment:
Don't need the class_exists as noted in another comment:

```suggestion
		$is_user_using_two_factor = Two_Factor_Core::is_user_using_two_factor();
```

Especially since if it doesn't fatal here, it will fatal further down the line.

Reply:
Ah yea, figured it was only needed there cause it hadn't been fataling where it was before. Updated